ios/RCCLightBox.m

```diff
@@ -48,10 +48,27 @@ -(instancetype)initWithFrame:(CGRect)frame params:(NSDictionary*)params
                 UIColor *backgroundColor = [RCTConvert UIColor:style[@"backgroundColor"]];
                 if (backgroundColor != nil)
                 {
-                    self.overlayColorView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, frame.size.width, frame.size.height)];
+                    self.overlayColorView = [[UIView alloc] init];
                     self.overlayColorView.backgroundColor = backgroundColor;
                     self.overlayColorView.alpha = 0;
+                    [self.overlayColorView setTranslatesAutoresizingMaskIntoConstraints:NO];
                     [self addSubview:self.overlayColorView];
+
```

Comment:
I think `self.overlayColorView.frame = self.bounds` in `layoutSubviews` should do the trick too.